tensornetwork/graphmode_tensornetwork_test.py

```diff
@@ -38,6 +39,65 @@ def test_gradient_decent(self):
       sess.run(train_op)
       self.assertLess(sess.run(final_tensor), 10.0)
 
+  def test_dynamic_network_sizes(self):
```

Comment:
Would it be a good idea to have the eager tests in a seperate class or change the name of this file since not everything is graph mode anymore?

Reply:
Yeah that's a good point. 

Since these are all tensorflow specific tests, I think it makes sense to move this to backends/tensorflow. Maybe call it `tensorflow_tensornetwork_test.py`?